inspirehep/modules/records/wrappers.py

```diff
@@ -19,16 +18,36 @@
 #
 # In applying this licence, CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization
-#}
-
-<div id="similar-jobs">
-  {% if record.rank %}
-    <b>{{ record.rank[0] }}</b>
-  {% endif %}
-  <br>
-  <a href="/jobs/{{record.control_number}}">{{ record.position }}</a>
-  <br>
-  {{ record.institution[0].name }}
-  <br>
-  Deadline: {{ record.deadline_date | format_date }}
-</div>
\ No newline at end of file
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from invenio_records.api import Record
+from invenio_search.api import RecordsSearch
+
+
+class JobsRecord(Record):
```

Comment:
This is a concept I wanted to have for quite some time. Note here how it neatly wraps the querying behaviour and delegates the presentation behaviour to the template, while previously the filter was doing both.